Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM ubuntu:22.04
```

Comment:
Can we use 24.04 or debian 12 here? My preference would be debian.

Reply:
I chose ubuntu 22.04 as it's the current LTS, but if you'd like another base image, I can change it (or you can do it yourself), it should "just work".

Reply:
should be fine for now atleast.